src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpCodeParser.cs

```diff
@@ -39,11 +39,16 @@ internal class CSharpCodeParser : TokenizerBackedParser<CSharpTokenizer, CSharpS
         private Dictionary<CSharpKeyword, Action<bool>> _keywordParsers = new Dictionary<CSharpKeyword, Action<bool>>();
 
         public CSharpCodeParser(ParserContext context)
+            : this (directiveDescriptors: Enumerable.Empty<DirectiveDescriptor>(), context: context)
```

Comment:
I don't think there's any value in making the "built in" directives **unparsable** by removing them. Users have the ability to make "built in" directives invalid in the IR if they don't want them to be supported.